chroot_setup.sh

```diff
@@ -522,6 +522,20 @@ for chroot in ${mylist[@]}
   #Clean Up 
   rm $CHROOT_TMP
 }
+mkdir -p "$CHROOT_DIR/QOpenSys/etc"
+ETC_PROFILE="$CHROOT_DIR/QOpenSys/etc/profile"
+if [ ! -f "$ETC_PROFILE" ]; then
+  echo "Priming /QOpenSys/etc/profile"
+  cat <<EOF >> "$ETC_PROFILE"
+PATH=/QOpenSys/pkgs/bin:$PATH
+export PATH
+OBJECT_MODE=64
+export OBJECT_MODE
+EOF
+fi
+
 printf "\nTo enter Your Chroot"
 printf "\nRUN: chroot $CHROOT_DIR /QOpenSys/usr/bin/sh\n"
+printf "\nTo set up your PATH to pick up RPM packages once inside your chroot"
+printf "\nRUN: source /QOpenSys/etc/profile"
```

Comment:
@ThePrez 

I tested this out the .profile was created successfully.

Although entering the chroot with  `chroot $CHROOT_DIR /QOpenSys/usr/bin/sh` and running `source /QOpenSys/etc/profile` errored out with:

`/QOpenSys/usr/bin/sh: source:  not found`

Using the dot operator `.  /QOpenSys/etc/profile` worked successfully though.

We may want to recommend using the dot operator here should work with both `ksh` and `bash`